deployment.rb

```diff
@@ -212,16 +212,19 @@ def firebase_channel_id_suffix
     ''
   end
 
-  def site_url(domain, path = '', scheme = '')
+  def site_host(domain)
     host = canonical_hostname(domain)
     if (rack_env?(:development) && !CDO.https_development) ||
-        (ENV['CI'] && host.include?('localhost'))
+      (ENV['CI'] && host.include?('localhost'))
       port = ['studio.code.org'].include?(domain) ? CDO.dashboard_port : CDO.pegasus_port
       host += ":#{port}"
     end
+    host
+  end
```

Comment:
Extracted a helper because runner.rb wants the hostname w/port _without_ the leading `//`.